packages/opentelemetry-context-zone/package.json

```diff
@@ -67,7 +67,7 @@
     "webpack-merge": "5.7.3"
   },
   "dependencies": {
-    "@opentelemetry/context-zone-peer-dep": "0.19.0",
+    "@opentelemetry/context-zone-peer-dep": "^0.20.0",
```

Comment:
Seems like we pin sometimes but not everywhere.

Reply:
Nice catch on this one (I thought I went through all of them (as per comment above)) -- this one *should* be pinned as it's a package